core/sqf/sqenvcom.sh

```diff
@@ -29,8 +29,8 @@
 #  * Add an entry in sqf/LocalSettingsTemplate.sh to make path configurable.
 ##############################################################
 
-#Product version (Trafodion or derivative product)
-export TRAFODION_VER_PROD="Apache Trafodion "
+#Product version (Trafodion, Enterprise or Enterprise Adv)
```

Comment:
"Enterprise", "Enterprise Adv" sound like product names. Just curious why we changed this away from "deriviative product"?